reference/cache.md

```diff
@@ -69,6 +68,16 @@ And the following routes are **not** cached:
 > **note**
 > Regular expressions in routes are **not** supported.
 
+## Cache behavior
+
+Cache is only applied to `GET` and `HEAD` requests.
+Responses with `Cache-Control` header set to `Private`, `No-Cache`,
+or `No-Store` are not cached. Also, this applies to responses with
+`Set-Cookie` header being set.
+
+Cache would also respect the `Vary` header in the response.
+Multiple versions of the cache will be created if `Vary` header is set.
```

Comment:
`Vary` is similar to cache key. See https://www.nginx.com/blog/nginx-plus-r5-released/.